cli/main.go

```diff
@@ -1,3 +1,5 @@
+//go:generate go run static/generate.go
```

Comment:
`go:generate` is relative to the directory of the file it is annotated in. Therefore, `static/generate/.go` is valid.